cloud/disk_manager/deploy/helm/disk-manager/templates/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata: {{ $fullname := printf "%s-%s" "disk-manager" .Values.serviceKind }}
+  name: {{ $fullname }}
+  namespace: {{ .Values.namespace | default "disk-manager" }}
+  labels:
+    app: {{ $fullname }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+  annotations:
+    checksum/client-config: {{ tpl (.Files.Get "configs/client-config.txt") . | sha256sum }}
+    checksum/server-config: {{ tpl (.Files.Get "configs/server-config.txt") . | sha256sum }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ $fullname }}
+  replicas: {{ default 9 .Values.replicaCount | int }}
+  template:
+    metadata:
+      labels:
+        app: {{ $fullname }}
+    spec: {{ $serverConfigName := "server-config.txt" }}
+      {{ $clientConfigName := "client-config.txt" }}
+      {{ $serverConfigPath := printf "/etc/yc/disk-manager/%s" $serverConfigName}}
+      {{ $clientConfigPath := printf "/etc/yc/disk-manager/%s" $clientConfigName}}
+      volumes:
+      - name: disk-manager-config
+        configMap:
+          name: "disk-manager-config-{{ .Values.serviceKind }}"
+      initContainers:
+      - name: disk-manager-init
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+        command: ["/usr/bin/yc-disk-manager-init-db", "--config={{ $serverConfigPath }}"]
+        volumeMounts:
+        - name: disk-manager-config
+          mountPath: "{{ $serverConfigPath }}"
+          subPath: server-config.txt
+
+      containers:
+      - name: disk-manager
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        command: ["/usr/bin/yc-disk-manager", "--config={{ $serverConfigPath }}"]
+        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+        volumeMounts:
+        - name: disk-manager-config
+          mountPath: "{{ $serverConfigPath }}"
+          subPath: "{{ $serverConfigName }}"
+        {{- if eq .Values.serviceKind "controlplane" }}
+        - name: disk-manager-config
```

Comment:
if serviceKind is controlplane you will have two volumeMounts with the same name `disk-manager-config` (line 45, 49 are the same)

Reply:
It works though due to different subPaths.